tardis/energy_input/indivisible_packets.py

```diff
@@ -313,10 +315,11 @@ def main_gamma_ray_loop(num_decays, model):
     co56_lines = read_artis_lines("co56")
 
     total_energy = (
-        activity_df["Ni56"] * ni56_lines.energy * 1000 * ni56_lines.intensity
-    ).sum() + activity_df["Co56"] * (
-        co56_lines.energy * 1000 * co56_lines.intensity
-    ).sum()
+        activity_df["Ni56"]
```

Comment:
Make sure you have a comment describing the unit conversion each time you multiply by one of the 1000s (I know it should be obvious, but best to document all code magic numbers)